addons/event_booth_sale/models/sale_order_line.py

```diff
@@ -38,11 +38,25 @@ def _compute_event_booth_pending_ids(self):
 
     def _inverse_event_booth_pending_ids(self):
         for so_line in self:
+            existing_booths = so_line.event_booth_registration_ids.event_booth_id
+            selected_booths = so_line.event_booth_pending_ids
+
+            booths_to_create = selected_booths\
+                .filtered(lambda booth: booth not in existing_booths) if existing_booths else selected_booths
```

Comment:
Linting: I prefer this instead (not a fan of `\` :slightly_smiling_face:)
```suggestion
            booths_to_create = selected_booths.filtered(
                lambda booth: booth not in existing_booths) if existing_booths else selected_booths
```